### PR TITLE
Updated CodeShip, added Jenkins X and Sentry

### PR DESCRIPTION
**Pull request by moritzplassnig**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,9 +46,10 @@ A curated list of awesome companies that offer their tools and services for free
 - [AppVeyor](https://www.appveyor.com/) - CI/CD for Windows developers.
 - [CircleCI](https://circleci.com/) - Docker-based building with support of customized workflows.
 - [Codefresh](https://codefresh.io/) - Docker-native CI/CD.
-- [Codeship](https://codeship.com/) - Continuous integration, delivery, and deployment.
+- [CloudBees CodeShip](https://codeship.com/) - Continuous integration, delivery, and deployment.
 - [continuousphp](https://continuousphp.com/) - CI/CD for PHP applications.
 - [DeployHQ](https://www.deployhq.com/) `requires-approval` - Deployment automation.
+- [Jenkins X](https://jenkins-x.io/) - CI/CD for Kubernetes.
```
Comment (zachflower) [inline README.md:52]: I can't tell from their website, is Jenkins X a commercial product?

File: README.md [hunk 2]
```diff
@@ -73,6 +74,7 @@ A curated list of awesome companies that offer their tools and services for free
 
 - [BugSnag](https://www.bugsnag.com/) `requires-approval` - Exception tracking and reporting.
 - [Honeybadger](https://www.honeybadger.io) `requires-approval` `non-commercial` - Exception and uptime monitoring for Ruby.
+- [Sentry](https://sentry.io) - Exception tracking and reporting
```
Comment (zachflower) [inline README.md:77]: Sentry as a product is open source, but as far as I can tell they don't actually offer any specific benefits to other open source projects. This usually comes in the form of specific, open source pricing (which is separate from a free tier that anyone can use). Does Sentry offer any price breaks or benefits to other open source projects?

File: README.md (hunk 1 above)
Comment (moritzplassnig) [reply README.md:52]: It's open source, see here: https://github.com/jenkins-x/jx/blob/master/LICENSE

File: README.md (hunk 2 above)
Comment (moritzplassnig) [reply README.md:77]: No. If that's the criteria, I misunderstood. I assumed that offering it as OSS also counts as "OSS supporter"

Comment (moritzplassnig) [reply README.md:77]: @zachflower Do you want me to update the PR and remove Sentry?

**Pull request closed without merging**